universite-du-quebec-a-montreal.csl

```diff
@@ -82,15 +111,20 @@
         <choose>
           <if type="report">
             <text variable="publisher"/>
-            <text macro="title"/>
+            <group suffix=".">
+              <text variable="title" font-style="italic"/>
+            </group>
           </if>
         </choose>
       </substitute>
     </names>
+    <names variable="composer">
+      <name and="text" delimiter-precedes-last="never" et-al-min="8" et-al-use-first="6" et-al-subsequent-min="8" et-al-subsequent-use-first="6" et-al-use-last="true" initialize-with="." name-as-sort-order="all"/>
```

Comment:
we prefer the et al settings in the bibliography/citation line unless you really need to differentiate et al within citations or the bibliography.

Reply:
Understood.